tla/consensus/Traceccfraft.tla

```diff
@@ -77,12 +77,12 @@ IsMessage(msg, dst, src, logline) ==
 ASSUME TLCGet("config").mode = "bfs"
 
 JsonFile ==
-    IF "JSON" \in DOMAIN IOEnv THEN IOEnv.JSON ELSE "../traces/consensus/append.ndjson"
+    IF "DRIVER_TRACE" \in DOMAIN IOEnv THEN IOEnv.DRIVER_TRACE ELSE "../traces/consensus/append.ndjson"
 
 JsonLog ==
     \* Deserialize the System log as a sequence of records from the log file.
     \* Run TLC from under the tla/ directory with:
-    \* $ JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccfraft.tla
+    \* ./tlc.py --driver-trace ../build/startup.ndjson consensus/Traceccfraft.tla
```

Comment:
TV doesn't distinguish the source of a trace; it could be either from a driver, some e2e test, or even a production environment.  Consider removing the source from the parameter name.